src/common/hotbar-store/store.ts

```diff
@@ -30,7 +31,9 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
       syncOptions: {
         equals: comparer.structural,
       },
-      migrations,
+      migrations: ipcMain
+        ? asLegacyGlobalObjectForExtensionApi(hotbarStoreMigrationsInjectable)
```

Comment:
If this is indeed required, it should be created with other similar globals, instead of inline.